src/Theme/Global.elm

```diff
@@ -178,6 +180,16 @@ globalStyles =
 -- Helpers
 
 
+primaryHeader : String -> Html Msg
+primaryHeader title =
+    h1 [ css [ primaryHeaderStyle ], id "focus-target" ] [ text title ]
+
+
+primaryHeaderStyle : Style
+primaryHeaderStyle =
+    fontFamilies [ "Ludicrous", "serif" ]
```

Comment:
I don't think this is right - I think this font used to be used on the guide and story pages but not any of the index pages, and it looks like it's not in use on any of the headers any more https://miro.com/app/board/uXjVPyf-Msw=/

Reply:
I'm also wondering if this style is necessary - we already apply styling to all h1 tags globally, so I think you could just remove this

Reply:
That feels bad to me: having an inconsistent style on pages.

Reply:
What do you mean? All the pages have the same style now, they just used to have a different one - if you have. alook at the miro you can see what I mean! They just don't have this font - they should just have the styles being applied in theme.global to the h1 tags automatically

Reply:
no problem, removed local styling